chrome/content/coniks/context.js

```diff
@@ -15,15 +13,19 @@ class ConiksContext {
     }
     this.account = opts.account;
     // user policies
-    this.allowUnsignedKeyChange = opts.allowUnsignedKeyChange === null || opts.allowUnsignedKeyChange === undefined ?
-                                  true : opts.allowUnsignedKeyChange;
-    this.allowPublicLookup = opts.allowPublicLookup === null || opts.allowPublicLookup === undefined ?
-                             true : opts.allowPublicLookup;
+    this.signedKeyChange =
+      opts.signedKeyChange === null ||
+      opts.signedKeyChange === undefined ||
+      opts.signedKeyChange;
```

Comment:
If you're doing this, you only want to do,

```
this.signedKeyChange = !!opts.signedKeyChange;
```

and update how this value is used elsewhere.